### DISCUSSION TITLE
dandi metadata: assets summary

### DISCUSSION THREAD

**Discussion by bendichter** [dandi/helpdesk#18] [Q&A] [answered]
The Assets Summary does not make much sense for several dandisets. Is there a way of re-generating this, manually correcting it, or just removing it?

![image](https://user-images.githubusercontent.com/844306/124324377-d9085280-db50-11eb-9fe2-f9e2de90217e.png)

**Comment by yarikoptic** [comment]
I think this should be fixed on dandi-api side  by making removing assetsSummary on dandisets which have no assets. Attn @dchiquito

**Comment by satra** [comment, marked as answer]
This is an outcome of the current hack to suppress summary generation. Should revert back once the updates are in place.

**Reply by yarikoptic** [reply]
What will happen then for dandisets without assets?

**Reply by satra** [reply]
should still have a summary of 0 files and bytes, just won't be publishable with that summary.

**Reply by yarikoptic** [reply]
FTR: https://github.com/dandi/dandi-api/pull/389 is filed. Inquiring on the best way to trigger updates across dandisets there.
